lib/smbios.c

```diff
@@ -6,11 +6,14 @@
  */
 
 #include <dm.h>
+#include <dm/device-internal.h>
+#include <dm/lists.h>
 #include <env.h>
 #include <linux/stringify.h>
 #include <linux/string.h>
 #include <mapmem.h>
 #include <smbios.h>
+//#include <smbios_plat.h>
```

Comment:
Why is this commented?

Reply:
It should be clean. I will clean up all stuff related to coding style, currently it is a prototype.